### DISCUSSION TITLE
'CreateChild' method of DataPortal and Application server

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3123] [Questions] [answered]
In Using Csla 4 -Data Access book, we have "CreateChild method will never go across the network to an application server" (P116). 
I was wondering if it's still applicable? if so, how does Unit of work object help in accessing Application server? Does the DataPortal.Fetch in Unit of work object do the trick?

Kind Regards

**Comment by rockfordlhotka** [comment, marked as answer]
Yes, only the root data portal calls (Create/Fetch/Update/Execute) implement the mobile object concept. All child data portal calls are lightweight, don't clone, don't go remote, all they do is invoke the appropriate child data portal operation method in the current context.

The UOW pattern relies on the root object being invoked by the root data portal to be transported to the app server, and once there it can use the root or child data portal to implement all the server-side work.

**Reply by Chicagoan2016** [reply]
@rockfordlhotka  Thank you Sir.
